apps/process/views.py

```diff
@@ -47,9 +49,34 @@ def from_dashboard(request):
         'datensatzName': datensatzName,
         'erfassungsDatum': erfassungsDatum
         }
+
+    newExperiment = Experiment(name=datensatzName, timerow=zeitreihenSpalte, project_id=id)
+    newExperiment.save()
+
+    #if saveExperiment == 1:
+
+
+    saveExperiment(request, id, dataForRender)
+
     return render(request, "process/index.html", dataForRender)
 
 
+def saveExperiment(request, id, dataForRender):
+    jsonHeader = request.POST.get("jsonHeader", "")
+    jsonEinheiten = request.POST.get("jsonEinheiten", "")
```

Comment:
Englisch?

Reply:
Hab ich in Issue https://github.com/IT-PM-OpenAdaptronik/Webapp/issues/132 aufgenommen, dann machen wir das nächste Woche.